server/src/main/java/io/crate/execution/ddl/tables/DropPartitionsRequest.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Licensed to Crate.io GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.execution.ddl.tables;
+
+import java.io.IOException;
+import java.util.List;
+
+import org.elasticsearch.action.support.master.AcknowledgedRequest;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+
+import io.crate.metadata.RelationName;
+
+public class DropPartitionsRequest extends AcknowledgedRequest<DropPartitionsRequest> {
+
+    private final RelationName relationName;
+    private final List<String> partitionNames;
+
+    public DropPartitionsRequest(RelationName relationName, List<String> partitionNames) {
```

Comment:
I think this should either be a list of partition idents or a `List<List<String>>` for a list of partiton values.

The name of indices will merge with the uuid in the future (or at least become decoupled from the relation name) to support renaming relations without touching the linked indices/partitions

Reply:
thx! please check again.